### PR TITLE
[bug 1164091] Remove the last of cache machine.

### PR DESCRIPTION
**Pull request by rlr**
r?

### CODE REVIEW

File: scripts/travis/setup.sh [hunk 1]
```diff
@@ -17,7 +17,6 @@ DATABASES['default']['HOST'] = 'localhost'
 DATABASES['default']['USER'] = 'travis'
 DATABASES['default']['CONN_MAX_AGE'] = 600
 CELERY_ALWAYS_EAGER = True
-CACHE_BACKEND = 'caching.backends.locmem://'
```
Comment (rlr) [inline scripts/travis/setup.sh]: pretty sure this is an invalid (deprecated/remove) setting that isn't getting used anymore

Comment (rlr) [inline scripts/travis/setup.sh]: (we'll find out when travis pases or fails)

Comment (mythmon) [inline scripts/travis/setup.sh]: Yup. https://docs.djangoproject.com/en/1.4/ref/settings/#cache-backend

**Comment by mythmon** [comment]
r+

SMASH

**Comment by rlr** [comment]
great gif hahaha

**Pull request merged**